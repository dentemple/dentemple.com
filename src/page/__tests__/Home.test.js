import React from 'react'
import { shallow } from 'enzyme'
import MyComponent from '../Home'

it('renders without crashing', () => {
  shallow(<MyComponent />)
})

it('matches the snapshot', () => {
  const component = shallow(<MyComponent />)
  expect(component).toMatchSnapshot()
})
