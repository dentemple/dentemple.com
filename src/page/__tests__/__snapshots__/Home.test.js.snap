// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <HeroBackground />,
        <h2>
          Website
</h2>,
        <p>
          Lorem ipsum
</p>,
        <RaisedButton
          disabled={false}
          fullWidth={false}
          href="https://github.com/dentemple"
          label="Github Link"
          labelPosition="after"
          primary={false}
          secondary={true}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Website",
        },
        "ref": null,
        "rendered": "Website",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem ipsum",
        },
        "ref": null,
        "rendered": "Lorem ipsum",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "disabled": false,
          "fullWidth": false,
          "href": "https://github.com/dentemple",
          "label": "Github Link",
          "labelPosition": "after",
          "primary": false,
          "secondary": true,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <HeroBackground />,
          <h2>
            Website
</h2>,
          <p>
            Lorem ipsum
</p>,
          <RaisedButton
            disabled={false}
            fullWidth={false}
            href="https://github.com/dentemple"
            label="Github Link"
            labelPosition="after"
            primary={false}
            secondary={true}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Website",
          },
          "ref": null,
          "rendered": "Website",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem ipsum",
          },
          "ref": null,
          "rendered": "Lorem ipsum",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "disabled": false,
            "fullWidth": false,
            "href": "https://github.com/dentemple",
            "label": "Github Link",
            "labelPosition": "after",
            "primary": false,
            "secondary": true,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
